README.md

```diff
@@ -563,12 +655,53 @@ sudo systemctl reload nginx
 
 At this step, you should also setup the SSL for Nginx. [Certbot](https://certbot.eff.org/lets-encrypt/ubuntuxenial-nginx) can be a good option if you want a free SSL certificate.
 
+
+### Apache
+
+Install Apache and make sure to replace `mydomain.com` by your domain
+
+```bash
+sudo apt-get install -y apache2
+```
+```
+<IfModule mod_ssl.c>
+  <VirtualHost _default_:443>
+    ServerName mydomain.com
+    DocumentRoot /var/www/html
+    ErrorLog ${APACHE_LOG_DIR}/error.log
+    CustomLog ${APACHE_LOG_DIR}/access.log combined
+    SSLEngine on
+    SSLCertificateFile      /etc/ssl/certs/ssl-cert-snakeoil.pem
+    SSLCertificateKeyFile /etc/ssl/private/ssl-cert-snakeoil.key
+    ProxyPreserveHost On
+    ProxyPass / http://127.0.0.1:7777/
+    ProxyPassReverse / http://127.0.0.1:7777/
+    <FilesMatch "\.(cgi|shtml|phtml|php)$">
+      SSLOptions +StdEnvVars
+    </FilesMatch>
+    <Directory /usr/lib/cgi-bin>
+      SSLOptions +StdEnvVars
+    </Directory>
+  </VirtualHost>
+</IfModule>
+```
+
+this will use the default self-signed cert.
+
 ### Enjoy!
 
 If all of the above steps are successful, open http://app.mydomain.com/ and create your first account!
 
-By default, new accounts are not premium so don't have unlimited alias. To make your account premium,
-please go to the database, table "users" and set "lifetime" column to "1" or "TRUE".
+By default, new accounts are **not premium** so don't have unlimited alias. To make all accounts premium by default,
+use the following commands:
+
+`sudo docker exec -it sl-db psql -U myuser simplelogin`
```

Comment:
I wonder if we can group these into a single command, something like the below. I haven't tested the command so this is just for idea :).

```
echo "ALTER TABLE users ALTER COLUMN lifetime SET DEFAULT TRUE;" | sudo docker exec -it sl-db psql -U myuser simplelogin
```

Reply:
echo "ALTER TABLE users ALTER COLUMN lifetime SET DEFAULT TRUE;" | sudo docker exec -it sl-db psql -U myuser simplelogin
responds with:
the input device is not a TTY